security/pkg/pki/ca/ca.go

```diff
@@ -160,7 +162,7 @@ func NewSelfSignedIstioCAOptions(ctx context.Context, caCertTTL, certTTL, maxCer
 
 		options := util.CertOptions{
 			TTL:          caCertTTL,
-			Org:          org,
+			Org:          fmt.Sprintf("%s (%s)", org, uuid.New().String()),
```

Comment:
Ok for now - in future we may want to set it based on a 'cluster' flag ( if specified, and default to UUID otherwise).

K8S CA is using UUID.